examples/Makefile

```diff
@@ -310,7 +310,7 @@ ifeq ($(PLATFORM),PLATFORM_DESKTOP)
     ifeq ($(PLATFORM_OS),LINUX)
         # Libraries for Debian GNU/Linux desktop compiling
         # NOTE: Required packages: libegl1-mesa-dev
-        LDLIBS = -lraylib -lGL -lm -lpthread -ldl -lrt
+        LDLIBS = -lraylib -lGL -lm -lpthread -ldl -lrt -latomic
```

Comment:
Its ONLY specific to RPI OS in desktop mode.
I haven't checked this in x86 Linux desktop.

Reply:
then I don't think you can do this, it will break compilation for those users
Not 100% sure on it, I will give it a go myself, but it looks sus

Reply:
Nevermind, I tried it and it still worked
In my case, on arch linux, /usr/lib/libatomic.so seems to be part of the package gcc-libs, which should be available for most users
Still not ideal to link a library that isn't necessary, just because it's there. I hope there's a better approach

Reply:
@Peter0x44 : The bigger question is, why doesn't Raylib 4.0.0 need latomic to compile?
When I try to run raylib games on piCore linux, its annoying when system says latomic is missing. The same doesn't happen for Raylib 4.0.0 games.

Reply:
I believe this is because the -latomic dependency on the RPI was introduced in a later version of miniaudio, that raylib 4.0 doesn't use

Reply:
I just dusted off my old Ubuntu 16 (64 bit) laptop and confirmed that -latomic wasn't needed there to compile.
So. This requirement seems like RPIOS Desktop only.

Reply:
I think one possible solution is checking for:
```c
#if defined(__arm__) || defined(__aarch64__)
    LDLIBS += -latomic
#endif
```

Reply:
From what I have gathered, -latomic is only needed on 32 bit systems.
I just confirmed this by trying to compile raylib on 64 bit bullseye and -latomic error didn't show up.

@raysan5 : Could you reject my PR and add this at your end?

Reply:
@Trithilon I'm actually checking how to add those lines equivalent to Makefile and I can't find a solution to properly detect architecture automatically on the Makefile...